packages/amazon-cognito-identity-js/__tests__/CognitoIdToken.test.js

```diff
@@ -0,0 +1,8 @@
+import CognitoIdToken from '../src/CognitoIdToken';
+
+describe('Constructor for CognitoId Token', () => {
+	test('Constructing a CognitoID Token', () => {
+		const cognitoToken = new CognitoIdToken('testToken');
+		expect(typeof cognitoToken.jwtToken).toBe('string');
```

Comment:
Nit: I think it's better to expect the actual value `''`. 
When checking `typeof` we don't know if maybe `testToken` actually got assigned to `jwtToken` somehow.

```suggestion
		expect(cognitoToken.jwtToken).toBe('');
```